include/odp/api/spec/ipsec.h

```diff
@@ -1715,6 +1786,32 @@ int odp_ipsec_result(odp_ipsec_packet_result_t *result, odp_packet_t packet);
  */
 int odp_ipsec_status(odp_ipsec_status_t *status, odp_event_t event);
 
+/**
+ * IPSEC test API for modifying internal state of an SA.
+ *
+ * This function is not meant to be used by normal applications but by special
+ * test applications that test or debug the operation of the underlying ODP
+ * implementation. Calling this function may degrade the performance of the
+ * calling thread, other threads or the IPSEC implementation in general.
+ *
+ * Calling this function for an SA at the same time when the SA is used for
+ * processing traffic or when the SA is being modified through other parts
+ * of IPSEC API may result in undefined behaviour.
+ *
+ * SA state update through this function may not be supported by all ODP
+ * implementations, ODP instances or SA instances or at every moment. This
+ * function may fail for unspecified reasons even when the capability
```

Comment:
After dicsussing with Petri, it would be good to replace "may fail" with "may return failure". The former may perhaps incorrectly imply that the function might fail in some fatal, unrecoverable way (like segfault).

Reply:
Ack.